docs/AboutUs.adoc

```diff
@@ -4,16 +4,15 @@
 :imagesDir: images
 :stylesDir: stylesheets
 
-AddressBook - Level 3 was developed by the https://ay1920s1-cs2103t-f13-2.github.io/main[F13] team. +
-_{The dummy content given below serves as a placeholder to be used by future forks of the project.}_ +
+$aveNUS was developed by the https://ay1920s1-cs2103t-f13-2.github.io/main[F13-2] team. +
 {empty} +
 We are a team based in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore].
 
 == Project Team
 
 === Roby Tanama
 image::robytanama.png[width="150", align="left"]
-{empty}[https://github.com/robytanama[github]] [<<johndoe#, portfolio>>]
+{empty}[Github](https://github.com/robytanama) | <<johndoe#, Portfolio>>
```

Comment:
This is an important change. It helps people to understand what we actually do in this project.